Civi/Token/TokenCompatSubscriber.php

```diff
@@ -70,8 +70,7 @@ public function onEvaluate(TokenValueEvent $e) {
     $e->getTokenProcessor()->context['hookTokenCategories'] = \CRM_Utils_Token::getTokenCategories();
 
     $messageTokens = $e->getTokenProcessor()->getMessageTokens();
-    $returnProperties = array_fill_keys($messageTokens['contact'] ?? [], 1);
-    $returnProperties = array_merge(\CRM_Contact_BAO_Query::defaultReturnProperties(), $returnProperties);
+    $returnProperties = $this->getReturnFields($messageTokens['contact']);
```

Comment:
This didn't turn out to be important - but IMHO we should switch to a more hard-coded returnProperties since the Query object is ... the query object & we really want to migrate to v4 over time